src/System.Private.ServiceModel/tests/Scenarios/Contract/Message/MessageContractCommon.4.1.0.cs

```diff
@@ -15,7 +15,6 @@ namespace MessageContractCommon
     public class MessageContractConstants
     {
         // CONSTANTS
-        public const string wrapperName = "CustomWrapperName";
```

Comment:
Nit: could you have just changed the constant here instead of removing it?  The rest of the tests use these constants to avoid duplicating strings inside the tests. Not blocking.